K2Bridge.Tests.UnitTests/Visitors/RangeAggregationVisitorTests.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+namespace UnitTests.K2Bridge.Visitors
+{
+    using System.Collections.Generic;
+    using global::K2Bridge.Models.Request.Aggregations;
+    using global::K2Bridge.Tests.UnitTests.Visitors;
+    using global::K2Bridge.Visitors;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class RangeAggregationVisitorTests
+    {
+        [TestCase(0, 800, 800, 2000, ExpectedResult = "union (_data | where dayOfWeek >= 0 and dayOfWeek < 800 | summarize wibble | extend ['key'] = '0-800'), (_data | where dayOfWeek >= 800 and dayOfWeek < 2000 | summarize wibble | extend ['key'] = '800-2000') | project-reorder * asc")]
```

Comment:
about this part:
"project-reorder * asc"

There is a place in the code where we say that we refer to column names both as index and also as name. (and we say we need to fix this)
Wouldn't re ordering might cause issues if referring to a column by index?

Reply:
Just for the Range aggregation, I need this re-order for our column indices to work. I agree we should probably switch to indexing the columns by name instead of indices, it would be more robust.

Reply:
what happens if you dont re-order?

Reply:
Because of the `extend` statement, the column with the range ends up last, instead of first. So, I re-order to put it first, which is what the response parser expects.